readthedocs/templates/projects/project_import.html

```diff
@@ -70,7 +70,16 @@
       </span>
 
       <h1>{% trans "Import a Repository" %}</h1>
-
+      {% block import-repo %}
```

Comment:
I think you should put your code inside 
```
{% if has_connected_accounts %}
    .....
{% endif %}
```

Reply:
ohk

Reply:
What should I put message in else condition ?

Reply:
just wrap the condition over your code
```
{% if has_connected_accounts %}
    <your code>
{% endif %}
```

Reply:
Hi @stsewd , Do I have to make more changes?